crates/turborepo-lib/src/run/mod.rs

```diff
@@ -175,10 +181,40 @@ impl<'a> Run<'a> {
 
         let is_single_package = opts.run_opts.single_package;
 
-        let mut pkg_dep_graph =
-            PackageGraph::builder(&self.base.repo_root, root_package_json.clone())
-                .with_single_package_mode(opts.run_opts.single_package)
-                .build()?;
+        // There's some warning handling code in Go that I'm ignoring
+        let is_ci_or_not_tty = turborepo_ci::is_ci() || !std::io::stdout().is_terminal();
+
+        let mut daemon = None;
+        if is_ci_or_not_tty && !opts.run_opts.no_daemon {
+            info!("skipping turbod since we appear to be in a non-interactive context");
+        } else if !opts.run_opts.no_daemon {
+            let connector = DaemonConnector {
+                can_start_server: true,
+                can_kill_server: true,
+                pid_file: self.base.daemon_file_root().join_component("turbod.pid"),
+                sock_file: self.base.daemon_file_root().join_component("turbod.sock"),
+            };
+
+            let client = connector.connect().await?;
```

Comment:
I believe this will show up in the merge-conflict fixup, but this shouldn't be an error and on main it no longer is.